docker/build-tools/Dockerfile

```diff
@@ -41,6 +41,7 @@ ENV HADOLINT_VERSION=v1.17.1
 ENV HELM_VERSION=v2.10.0
 ENV HUGO_VERSION=0.55.5
 ENV JUNIT_MERGER_VERSION=adf1545b49509db1f83c49d1de90bbcb235642a8
+ENV K8S_CODE_GENERATOR_VERSION=kubernetes-1.13.10
```

Comment:
Is there anything newer we can use?

Reply:
Also, could we move the "kubernetes-" part from the version string to be instead part of the component names below, so that the version string remains just a version string?

Reply:
I'd have to go through and retest everything.  I picked 1.13, because it seemed like a solid base and would allow it to work with older versions (i.e. we could generate apis for older versions of istio too).

Reply:
As for `kubernetes-,` sure